src/UserSettingsPage.cpp

```diff
@@ -507,7 +441,7 @@ UserSettingsPage::resizeEvent(QResizeEvent *event)
         else
                 sideMargin_ = static_cast<double>(event->size().width() - contentMinWidth) / 2.;
 
-        if (sideMargin_ < 40)
+        if (sideMargin_ < 70)
```

Comment:
This should really be the scrollbar width, but I'm not sure, how to get to it. We may need to keep an additional reference around, so that we can access it here?

Reply:
This is finally gone! :D